README.md

```diff
@@ -5,22 +5,15 @@ Orion stands as a powerful command-line tool/daemon designed for identifying reg
 ## Quick Start
 
 ### Podman
+
 ```bash
 $ podman build -f Dockerfile -t orion
-# Needed env vars.
-# ES/OpenSearch Server where the results live
- $ export ES_SERVER='my-opensearch.perf.com'
- # Version of OpenShift
- $ export version=4.19
- # Index where the benchmark data is stored
- $ export es_benchmark_index=ripsaw-kube-burner*
- # Index where you store the run metadata
- $ export es_metadata_index=perf_scale_ci*
- $ podman run --env-host orion orion cmd --config orion/examples/trt-external-payload-node-density.yaml --hunter-analyze
+$ podman run orion orion cmd --config examples/trt-external-payload-node-density.yaml --hunter-analyze --input-vars='{"version": "4.19"}' --es-server='https://my-opensearch.perf.com' --benchmark-index=ripsaw-kube-burner-* --metadata-index=perf_scale_ci* --lookback=15d
```

Comment:
I see the concerns that `input-vars` is addressing, but we keep in mind that this is a breaking change in CI before we cut a new release. Its as ENVs over there at the moment.

Reply:
It is mentioned in the PR description